src/main/java/io/jenkins/plugins/tuleap_api/client/authentication/OpenIDClientApi.java

```diff
@@ -0,0 +1,15 @@
+package io.jenkins.plugins.tuleap_api.client.authentication;
+
+import com.auth0.jwk.Jwk;
+
+import java.util.List;
+
+public interface OpenIDClientApi {
+    String DISCOVERY_API = "/.well-known/openid-configuration";
+
+    List<Jwk> getSigningKeys();
+
+    UserInfo getUserInfo(AccessToken accessToken);
```

Comment:
Is there a reason for this not to be in the UserInfoApi ?

Reply:
I finally decided to put all the open id query in the same interface.  And after discussion it's also ok for you